pages/common/pystun3.md

```diff
@@ -0,0 +1,16 @@
+# pystun3
+
+> Classic STUN client written in python.
+> More information: <https://github.com/talkiq/pystun3>.
+
+- Make a STUN request:
+
+`pystun3`
+
+- Make a STUN request and specify the stun server:
+
+`pystun3 --stun-host {{stun.1und1.de}}`
+
+- Make a STUN request and specify the source port:
+
+`pystun3 --source-port {{7932}}`
```

Comment:
```suggestion
`pystun3 --port {{7932}}`
```

I think you got confused with the argument placeholder and flag

Ref: https://github.com/talkiq/pystun3/blob/681b36ce4812714449dfbf3d2f5004a2f0615240/README.rst?plain=1#L58